Create CODEOWNERS

woodiertexas opened the pull request:
Implement code owners

woodiertexas committed:
Create CODEOWNERS

peaceheis commented on .github/CODEOWNERS:
```diff
@@ -0,0 +1,4 @@
+src/ @woodiertexas @TibiNonEst @JoostMSoftware @peaceheis
```
```suggestion
src/main/java/ @woodiertexas @TibiNonEst @JoostMSoftware @peaceheis
```

peaceheis replied:
I think that the we can confine that to just Java files, no?

woodiertexas replied:
that'd work tbh

woodiertexas replied:
Change implmented.

woodiertexas committed:
Update CODEOWNERS

fix directory stuff

woodiertexas committed:
Update CODEOWNERS

implement @peaceheis's suggestion

KirboSoftware reviewed (approved):
Seems good to me :+1: maybe in the future, define more dirs to people so we won't get over flooded with PR reviews if it's not our alley?

woodiertexas committed:
Update CODEOWNERS

Made "@DebuggyTeam/Worldgen" the code owner for the "/data/" and "/wonders/world/" directories.

violet-sh commented on .github/CODEOWNERS:
```diff
@@ -0,0 +1,4 @@
+/src/main/java/ @woodiertexas @TibiNonEst @JoostMSoftware @peaceheis
```
```suggestion
/src/main/java/ @DebuggyTeam/core 
```
LGTM but imo this should be a team so if things change that can be done through the team instead of the file.

woodiertexas committed:
Update .github/CODEOWNERS

Implement TibiNonEst's suggestion.

Co-authored-by: tibs <git@tibinonest.me>
